spring-cql/src/test/java/org/springframework/cassandra/test/integration/CassandraRule.java

```diff
@@ -310,7 +313,15 @@ private void setupConnection() {
 				port = properties.getCassandraPort();
 			}
 			cassandraPort = port;
-			cluster = new Cluster.Builder().addContactPoints(hostIp).withPort(port).build();
+
+			QueryOptions queryOptions = new QueryOptions();
+			queryOptions.setRefreshSchemaIntervalMillis(0);
+
+			cluster = new Cluster.Builder().addContactPoints(hostIp).//
```

Comment:
Adopted the dot-slash-slash style from the other [Spring Data](https://github.com/spring-projects/spring-data-mongodb/blob/8f5b334951aa85ac3ea0fcff84d5f66551737ce3/spring-data-mongodb/src/test/java/org/springframework/data/mongodb/core/DefaultBulkOperationsIntegrationTests.java#L157) modules to prevent long chains on reformat.